pre-docs/markdown.properties

```diff
@@ -0,0 +1,477 @@
+
+# ============================================================================ #
+#
+#   GENERATED PROPERTIES FILE FOR USE WITH THE DITA COMMAND
+#
+#   Lines in this file that start with a number sign '#' are comments.
+#
+#   To set a build parameter, remove the commenting '#' at the start of the line
+#   and specify the value for the parameter after the '=' sign, for example,
+#
+#   args.filter = my-filter.ditaval
+#
+#   Use the `dita` command with the `--propertyfile` option to use the build
+#   parameters specified in a properties file, for example:
+#
+#   dita --input=my.ditamap --format=html5 --propertyfile=my.properties
+#
+#   Build parameters in this file are grouped by transformation type.
+#   Supported parameter values are listed in brackets [] after each description,
+#   with an asterisk (*) indicating the default value where appropriate.
+#
+# ============================================================================ #
+
+
+# ============================================================================ #
+#                                   COMMON
+# ============================================================================ #
+
+# args.draft = no
+#                     Specifies whether the content of <draft-comment> and 
+#                     <required-cleanup> elements is included in the output.
+#                     [ no ]*
+#                     [ yes ]
+
+# args.figurelink.style = NUMBER
+#                     Specifies how cross references to figures are styled in 
+#                     output.
+#                     [ NUMBER ]
+#                     [ NUMTITLE ]
+#                     [ TITLE ]
+
+# args.filter = 
+#                     Specifies a filter file to be used to include, exclude, or 
+#                     flag content.
+
+# args.gen.task.lbl = YES
+#                     Specifies whether to generate headings for sections within 
+#                     task topics.
+#                     [ NO ]
+#                     [ YES ]
+
+# args.grammar.cache = yes
+#                     Specifies whether the grammar-caching feature of the XML 
+#                     parser is used.
+#                     [ no ]
+#                     [ yes ]*
+
+# args.input.dir = 
+#                     Specifies the base directory for your documentation project.
+
+# args.output.base = 
+#                     Specifies the name of the output file without file 
+#                     extension.
+
+args.rellinks = noparent

```

Comment:
is the carriage return intentional? It looks like its on the other uncommented line to.

Reply:
I'm not sure - this was copied from https://github.com/puppetlabs/nebula-docs/blob/master/markdown.properties - we may not need the file at all, or may need to make changes to it.

Reply:
If you'd prefer not to use the file, you can run the build with a couple of extra arguments. `-Dargs.rellinks=noparent -Dargs.gen.task.lbl=YES`